build/README.md

```diff
@@ -1,7 +1,7 @@
 # ODS-CI Container Image
 
 A [Dockerfile](Dockerfile) is available for running tests in a container.
-The latest build can be downloaded from: https://quay.io/odsci/ods-ci:latest
+The latest build can be downloaded from: https://quay.io/odsci/ods-ci:latest (<em>this image has been deprecated, a new one will be available soon. Please build your own image using the instructions below</em>)
```

Comment:
OPTIONAL: I would delete this entire line if you don't plan to push an image.  That will remove any questions when or if the image is going to be updated

Reply:
I think we will publish it. But anyway, yeah I can remove the line and when we publish the image we can restore it